### ISSUE TITLE
Old labels not showing on mini map

### ISSUE THREAD

**Issue by misaugstad** [ProjectSidewalk/SidewalkWebpage#2185]
Not sure what happened, but for me my old labels are not being shown on the mini map when I reload the page. New labels are still being saved, the markers just are not being loaded. And when I load the page, I get the following error once for every label it is trying to put on the mini map:
```
InvalidValueError: setPosition: not a LatLng or LatLngLiteral: in property lat: not a number
```

**Comment by misaugstad** [comment]
Okay, I only tested this on the Columbus test and prod servers on that day. Now it is a few days later, and I'm not getting this error on those servers. And I believe @ThatOneGoat tried to test it and couldn't reproduce. It's just weird because I could reproduce it multiple times that day, but haven't been able to reproduce since.

**Comment by misaugstad** [comment]
fixed via #2367 

This was a product of the endpoint for depth data being taken down, and so the lat/lng values were not being saved in the database. Then with no lat/lng, they weren't showing up on the mini map.
